Update istioctl docs (#7502)
istio/istio.io: content/en/docs/setup/install/istioctl/index.md

$ istioctl install
{{< /text >}}

> Note: `istioctl install` and `istioctl manifest apply` are exactly the same command - `install` was added in 1.6 as a
> simplification and `manifest apply` is deprecated and will be removed in 1.7.

Reviewer: ```suggestion
{{< tip >}}
Note that `istioctl install` and `istioctl manifest apply` are exactly the same command. In Istio 1.6, the simpler `install`
command replaces `manifest apply`, which is deprecated and will be removed in 1.7.
{{< /tip >}}
```

$ istioctl install --set profile=demo
{{< /text >}}

## Check what's installed

> `Istioctl` saves a copy of the CR that was used to install Istio into the cluster, called `installed-state`. You can

Reviewer 2: I believe `istioctl` should be lower case the same as with your Note above.

Author: Thanks, done.

> `istioctl` saves a copy of the CR that was used to install Istio into the cluster, called `installed-state`. You can
> inspect this CR if you ever lose track of what is installed in a cluster. This CR is used to perform checks in some
> `istioctl` commands and should not be removed.

Reviewer: ```suggestion
The `istioctl` command saves the `IstioOperator` CR that was used to install Istio in a copy of the CR named `installed-state`.
You can inspect this CR if you lose track of what is installed in a cluster.

The `installed-state` CR is also used to perform checks in some `istioctl` commands and should therefore not be removed.
```

> 

Reviewer 2: Does it make sense to have to command to output the `installed-state` CR?

Author: Probably too trivial...and the page is already really long.

## Generate a manifest before installation

You can generate the manifest before installing Istio using the `manifest generate`
sub-command.
For example, use the following command to generate a manifest for the `default` profile:

{{< text bash >}}
$ istioctl manifest generate > $HOME/generated-manifest.yaml
{{< /text >}}

> The generated manifest can be used to both inspect exactly what is installed, and track changes to the manifest
> over time. While the `IstioOperator` CR represents the full user configuration and is sufficient for tracking them,
> the output from `manifest generate` also captures any changes in the underlying charts and can be used where
> tracking of the actual installed resources is desired.

Reviewer: ```suggestion
The generated manifest can be used to inspect what exactly is installed as well as to track changes to the manifest
over time. While the `IstioOperator` CR represents the full user configuration and is sufficient for tracking it,
the output from `manifest generate` also captures possible changes in the underlying charts and therefore can be
used to track the actual installed resources.
```

> The output from `manifest generate` can also be used to install Istio using `kubectl apply` or equivalent. Note that

Reviewer: ```suggestion
The output from `manifest generate` can also be used to install Istio using `kubectl apply` or equivalent. However,
```

these alternative installation methods may not apply the resources with the same sequencing of dependencies as

> `istioctl install` and are not tested as part of a release.

Reviewer: ```suggestion
`istioctl install` and are not tested in an Istio release.
```
